crypto/bn/asm/x86_64-gf2m.pl

```diff
@@ -174,8 +174,8 @@
 .type	bn_GF2m_mul_2x2,\@abi-omnipotent
 .align	16
 bn_GF2m_mul_2x2:
-	mov	OPENSSL_ia32cap_P(%rip),%rax
-	bt	\$33,%rax
+	mov	OPENSSL_ia32cap_P+4(%rip),%eax
+	bt	\$1,%eax
```

Comment:
Bit of a context here. `bt` is actually unfavourable from performance viewpoint (as it doesn't fuse with following conditional jump instruction). `test` would be better, but 64-bit constants such as `1<<33` can't be encoded with any instruction but specific flavour of `mov`. That's why `bt` is chosen here if you view OPENSSL_ia32cap_P as 64-bit value. And as mentioned earlier, it being viewed as 64-bit value is kind of a legacy thing. It doesn't mean that legacy has to held on to, but the fact is that it's not result of overlook, but result of conscious assessment and confirmation that it does the job just fine.  But either way, *if* you look at second 32-bit element, then `test \$2`,%eax` would/should be preferred.